### DISCUSSION TITLE
Heading 1

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#317] [Component suggestions]
<img width="720" alt="visuele weergave van de heading 1 component" src="https://github.com/user-attachments/assets/4506249b-2cc1-4d2c-ac07-93e01aede856">

## Naam

Heading 1

## Doel
Hoofdkop van een pagina.

## Ook bekend als (niet verplicht)

Heading, Titel

## Gerelateerde componenten
[Heading](https://github.com/orgs/nl-design-system/discussions/220), [Heading 2](https://github.com/orgs/nl-design-system/discussions/318), [Heading 3](https://github.com/orgs/nl-design-system/discussions/319), [Heading 4](https://github.com/orgs/nl-design-system/discussions/320), [Heading 5](https://github.com/orgs/nl-design-system/discussions/321), [Heading 6](https://github.com/orgs/nl-design-system/discussions/322), [Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289)

## Onderzoek

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Heading 1 op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=heading+1)
